gui/qt/transaction_dialog.py

```diff
@@ -461,19 +464,22 @@ def set_i_text_has_selection(b):
         i_text.customContextMenuRequested.connect(self.on_context_menu_for_inputs)
         i_text.setFont(QFont(MONOSPACE_FONT))
         i_text.setReadOnly(True)
-        i_text.setTextInteractionFlags(i_text.textInteractionFlags() | Qt.LinksAccessibleByMouse | Qt.LinksAccessibleByKeyboard)
+        i_text.setTextInteractionFlags(i_text.textInteractionFlags() | Qt.LinksAccessibleByMouse | Qt.LinksAccessibleByKeyboard | Qt.TextSelectableByKeyboard)
```

Comment:
I kinda dislike that `Qt.TextSelectableByKeyboard` flag in that it ends up drawing a blinking cursor even when you are using the mouse.

But if you really really want to be able to keyboard navigate.. I guess we have no choice. :/

Reply:
Oh.. but in i_text it's ok. That's a text browser so.. it looks less bad. :) Nevermind!

Reply:
I'll see if there is a way to prevent this. Maybe we can set it based on how the control got focus.

Reply:
Eh.. don't bother. If you can figure it out for normal widgets like QLabel -- that would be nice. But don't kill yourself over it. It's fine.  Just a tiny pet peeve.

For QTextBrowser I think the blinking cursor looks good.

But for QLabel.. it's a bit weird. :)

Again -- don't kill yourself to figure this out. It's like super low priority in my mind...

Reply:
On Windows + Linux they both have a non-blinking cursor. I'm not sure what you are saying about `QLabel`, I didn't change any `QLabel`. Both `i_text` and `o_text` are `QTextBrowser`. I submitted another PR to set keyboard text selection based on focus reason or when keys are pressed.

Reply:
Yeah my comment wasn't related to your PR -- sorry. It was just a general complaint that when you make QLabel keyboard selectable, even when you use the mouse.. you get a strange blinking cursor.. in a QLabel!! 🗡 

Good to know that trick exists (your TextBrowserWithFilter* class).  

So.. don't mind me. I was just old man complaining. :)